projects/core/src/user/store/reducers/user-details.reducer.ts

```diff
@@ -1,24 +1,28 @@
-import * as fromUserDetailsAction from '../actions/user-details.action';
-import { UserDetailsState } from '../user-state';
 import { User } from '../../../occ/occ-models/index';
+import * as fromUserDetailsAction from '../actions/user-details.action';
 
-export const initialState: UserDetailsState = {
-  details: <User>{},
-};
+export const initialState: User = <User>{};
 
 export function reducer(
   state = initialState,
   action: fromUserDetailsAction.UserDetailsAction
-): UserDetailsState {
+): User {
   switch (action.type) {
     case fromUserDetailsAction.LOAD_USER_DETAILS_SUCCESS: {
-      const details: User = action.payload;
+      return action.payload;
+    }
```

Comment:
What is the purpose of this change in LOAD_USER_DETAILS_SUCCESS reducer?

Reply:
My guess would be when we load the user details for the first time, we store it under `user.details` in the state.